README.md

```diff
@@ -49,3 +49,27 @@ To display posts corresponding to a stream
 url : /streams/<int:pk>/posts/
 method : GET
 ```
+#### Fetch Posts by User
+To display posts corresponding to a user
+```
+url:/users/<int:pk>/posts
+method: GET
+```
+#### Bookmark/Unbookmark
+To bookmark a post or to unbookmark already existing bookmark
+```
+url:/bookmarks/create/
+method: POST
+```
```

Comment:
Under a separate Bookmarks heading.
Add parameters.
Also I think there should be separate views to create / remove a bookmark . @d1vyanshu

Reply:
Cannot think of any disadvantage as of now of keeping both under same view except of course it decreases the readability of code.